jvcl/install/JVCLInstall/Compile.pas

```diff
@@ -1224,7 +1224,7 @@ function TCompiler.CompileDelphiPackage(TargetConfig: ITargetConfig; Project: TP
           Files.Add(ExtractFileName(ChangeFileExt(Project.SourceName, '.dpk'))); // readd, dcc removes items from the Files list
           Win64xResult := Dcc(TargetConfig, Project, DccOpt, DebugUnits, Files, nil, True);
           if Win64xResult <> 0 then
-            Exit(Win64xResult);
+            Exit;
```

Comment:
Well, no, this breaks the feature as `Exit(Win64xResult)` sets the return value of the function to `Win64xResult`

Reply:
And now this exits every time, not just when `Win64xResult` is different from 0 like before